src/components/Input/Input.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+const StyledInput = styled.input`
```

Comment:
```suggestion
const Input = styled.input`
```
I would also rename this to `Input` then.